src/Domain/Task/Handler/RescheduleHandler.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace AppBundle\Domain\Task\Handler;
+
+use AppBundle\Domain\Task\Command\Reschedule;
+use AppBundle\Domain\Task\Event;
+use AppBundle\Entity\Task;
+use SimpleBus\Message\Recorder\RecordsMessages;
+
+class RescheduleHandler
+{
+
+    public function __construct(private RecordsMessages $eventRecorder)
+    { }
+
+    public function __invoke(Reschedule $command)
+    {
+        $task = $command->getTask();
+        $rescheduledAfter = $command->getRescheduleAfter();
+        $rescheduledBefore = $command->getRescheduledBefore();
+
+        $this->eventRecorder->record(new Event\TaskRescheduled($task, $rescheduledAfter, $rescheduledBefore));
+
+        $task->setAfter($rescheduledAfter);
+        $task->setBefore($rescheduledBefore);
+        $task->unassign();
```

Comment:
I'm not sure this makes sense... just in terms of UI, the task will "jump" from the assigned panel to the unassigned panel, and it's easy to lose track of the task when dealing with a dashboard full of tasks.